javascript/ql/lib/change-notes/2024-11-28-regexp-unknown-flags.md

```diff
@@ -0,0 +1,5 @@
+---
+category: majorAnalysis
+---
+* Queries such as `IncompleteSanitization`, `TaintedPathCustomizations`, and `IncompleteBlacklistSanitizer` are now compatible with both `RegExpLiteral` and `RegExpObject`.
```

Comment:
Maybe focus more on the change that users will actually see.  
The change they'll see is that the queries now flag `new RegExp` objects, and not just regex literals. 

Same with the below.